RELEASE_NOTES.md

```diff
@@ -37,6 +37,11 @@ Release 2.8 introduces a new feature, `SpanLimit`, which provides a third way to
 
 Suppose, for example, that a service generates a single trace with 10,000 spans. If SpanLimit is set to 1000, once the first 1000 spans have arrived, Refinery will immediately make a decision to keep or drop the trace. Every additional span is dispatched (using the same decision) without storing it. This means that Refinery never had to keep all 10,000 spans in its memory at one time.
 
+### `In` and `NotIn` Operators in Rules
```

Comment:
There's a discrepancy in how we document here versus in the PR: https://github.com/honeycombio/refinery/pull/1302/files#diff-adf5ae21a5e738c1aab5407a6a4c83df85668d66363f27ccaea4788fba3e178bR672-R673

Reply:
In the below text, and the rules-complete file, the operator is shown as `In`/`NotIn`. But in the yaml file I link to, it's shown as `in`/`not-in`. Which is the operator? Or is the rulesmeta file just an inside baseball type representation?

Reply:
excellent catch. Thank you. The operator is `in` and `not-in` and I will fix those bugs!